test/integration/scripting_spec.mjs

```diff
@@ -2510,4 +2510,42 @@ describe("Interaction", () => {
       );
     });
   });
+
+  describe("Calculate field value even if one callback throws", () => {
+    let pages;
+    let otherPages;
+
+    beforeAll(async () => {
+      otherPages = await Promise.all(
+        global.integrationSessions.map(async session =>
+          session.browser.newPage()
+        )
+      );
+      pages = await loadAndWait("issue18561.pdf", getSelector("24R"));
+    });
+
+    afterAll(async () => {
+      await closePages(pages);
+      await Promise.all(otherPages.map(page => page.close()));
+    });
+
+    it("must check that the product are null", async () => {
```

Comment:
This description doesn't seem to reflect what's being tested here; shouldn't it be something like `must continue field value calculation on callback errors` or so?